Source/WebInspectorUI/UserInterface/Views/GradientSlider.js

```diff
@@ -414,8 +404,10 @@ WI.GradientSliderKnob = class GradientSliderKnob extends WI.Object
             x = Math.min(Math.max(0, x), w);
         this.x = x;
 
-        if (this._detaching)
+        if (this._detaching) {
             this.y = event.pageY - this._startMouseY;
```

Comment:
again, you need to change this to `this._y` since that's what's used by `WI.GradientSliderKnob.prototype._updateTransform`